mautic/mautic
# Improve APCu support

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | Fixes #8208
| BC breaks? | No
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
There were a few APC-specific functions that will not work in PHP 7 where APCu has supplanted those methods. In addition, when available APCu caching should be used for metadata and query caching (not results) for a small bump in performance.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. You should be able to log in, and edit a contact.
3. Save changes and refresh and see your changes to the contact.
